src/main/charts/confluence/templates/statefulset-synchrony.yaml

```diff
@@ -22,6 +22,20 @@ spec:
     spec:
       serviceAccountName: {{ include "confluence.serviceAccountName" . }}
       terminationGracePeriodSeconds: {{ .Values.synchrony.shutdown.terminationGracePeriodSeconds }}
+      {{- if .Values.confluence.securityContextEnabled }}
+      {{- with .Values.confluence.securityContext }}
```

Comment:
I am wonder if we should add a separated security context for Synchrony or we should re-use the confluence security context for Synchrony

Reply:
I am not sure using different security contexts for Synchrony and Confluence is the use case we'll see much. That's why I decided to unify them not to bloat values.yaml.

Reply:
Thanks, @bianchi2. Based on our previous experience it is better to provide complete customizability for the user as it is hard to predict how they will use the helm charts. I agree that adding this to the helm chart is a good idea. In general, the guiding principle is to provide a high degree of flexibility and don't worry about the size of `values.yaml` file.